components/Navigation/Navigation.module.scss

```diff
@@ -1,16 +1,73 @@
 .container {
   display: flex;
   font-family: Inter;
-
-  @media (max-width: 390px) {}
-
+  font-weight: 600;
+  color: #9c9eab;
+  justify-content: space-between;
+  @media (min-width: 320px) {
+    margin: 0 auto;
+  }
+  .text {
+    font-family: Assistant;
+    font-size: 2vw;
+    font-style: normal;
+    font-weight: 700;
+    line-height: normal;
+  }
   .nav {
-    @media (max-width: 390px) {
-      display: flex;
-      gap: 8px;
-      font-size: 16px;
-      font-weight: 600;
-      color: #9C9EAB;
+    display: flex;
+    gap: 8px;
+    margin: 0 8px 0 0px;
+    color: #9c9eab;
+    font-size: 2rem;
+    white-space: nowrap;
+    font-weight: 600;
+    gap: 8px;
+    @media (min-width: 320px) {
+      margin: 0;
+      font-size: 1rem;
+      gap: 3rem;
+    }
+    @media (min-width: 390px) {
+      margin: 0 2rem 0 0;
+      font-size: 1.6rem;
+      gap: 2rem;
+    }
+    @media (min-width: 450px) {
+      font-size: 1.6rem;
+      gap: 3rem;
+    }
+    @media (min-width: 576px) {
+      font-size: 1.6rem;
+      gap: 4.5rem;
+    }
+    @media (min-width: 768px) {
+      margin: 0 2rem 0 0;
+      font-size: 1.6rem;
+      gap: 1rem;
+    }
+    @media (min-width: 1024px) {
+      margin: 0 5.6rem 0 0px;
+      font-size: 2rem;
+    }
+    @media (min-width: 1440px) {
+      margin: 0 8.6rem 0 0;
+    }
+    .link {
+      // &:nth-child(-n + 2) {
```

Comment:
Don't keep code as comments. 

In case you foresee these lines to be required for a specific task that you are about to do next, 
add a `//TODO` item with a specific Github issue number and who you suppose will do it, like

`//TODO uncomment and use @habdevs #188`